mod2/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,12 @@
+# ������� �����������
+log4j.rootLogger=INFO, file
+
+# ������� ��� ������ � �������
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+# ���� ��� ����� ����������� ��� ����
+log4j.appender.file.File=C:\\Users\\Dmitry\\IdeaProjects\\JavaUpSkill\\mod2\\src\\log\\log_file.log
```

Comment:
Can we use relative path for file? Also log file should not be placed to code path